index.html

```diff
@@ -80,6 +78,21 @@
             </div>
         </section>
         <section>
+          <h2>A Note on Leadership: <br>We are an anarchist ghost pirate ship.</h2>
+          <p>What does this mean for you?</p>
+          <ul>
+            <li>We have very little formal structure and believe in consensus when possible.</li>
+            <li>You are empowered to make things happen (but ask for consensus).</li>
+            <li>We DO NOT vet the things people announce ahead of time. Do your research.</li>
+            <li>We DO have and enforce community standards through our code of conduct.</li>
```

Comment:
Grammar is slightly awkward:  the third item can be parsed as ( we do not vet ( the things people announce ahead of time ) ).  Maybe swap the last two items so that the first establishes context for the second, like:

* We have and enforce community standards through our code of conduct. This code covers our mailing list, Slack, wiki, and our monthly meetings.
* Other spaces and events, including those that may be announced here, may have their own rules and norms.

I'm neutral on "do your research", "do your homework", "caveat emptor", "YMMV", or similar sentiments.